src/Gui/CommandView.cpp

```diff
@@ -254,14 +254,17 @@ class StdCmdFreezeViews : public Gui::Command
 public:
     StdCmdFreezeViews();
     virtual ~StdCmdFreezeViews(){}
-    const char* className() const
+    const char* className() const override
     { return "StdCmdFreezeViews"; }
 
+    void setShortcut (const QString &) override;
+    QString getShortcut() const override;
+
 protected:
-    virtual void activated(int iMsg);
-    virtual bool isActive(void);
-    virtual Action * createAction(void);
-    virtual void languageChange();
+    virtual void activated(int iMsg) override;
+    virtual bool isActive(void) override;
+    virtual Action * createAction(void) override;
+    virtual void languageChange() override;

```

Comment:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final
```suggestion
   void activated(int iMsg) override;
    bool isActive(void) override;
    Action * createAction(void) override;
    void languageChange() override;

```